src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -422,10 +423,14 @@ export const Checkout = ({
         return
       }
     } catch (error) {
-      if (
-        !isManualReviewRequired((error.graphQLErrors || []) as GraphQLError[])
-      ) {
+      const isManualReviewRequired = checkIsManualReviewRequired(
+        (error.graphQLErrors || []) as GraphQLError[],
+      )
+      if (isManualReviewRequired) {
+        setIsManualReviewRequired(isManualReviewRequired)
         setIsShowingFailModal(true)
+        setSignUiState('MANUAL_REVIEW')
+        return
       }
```

Comment:
Do we need three separate pieces of state for this?

Reply:
Probably not LOL 🙈 Let's look into that later.